dbcrossbarlib/src/tokio_glue.rs

```diff
@@ -13,11 +14,11 @@ use crate::common::*;
 /// blocked on `.await`!), and we `Pin<Box<...>>` it using `.boxed()` to make it
 /// an abstract, heap-based type, for convenience. All we know is that it will
 /// return a `Result<T>`.
-pub type BoxFuture<T> = Pin<Box<dyn StdFuture<Output = Result<T>> + Send + 'static>>;
+pub type BoxFuture<T, E = Error> = futures::future::BoxFuture<'static, Result<T, E>>;
 
 /// A stream of values of type `T`, using our standard error type, and imposing
 /// enough restrictions to be able send streams between threads.
-pub type BoxStream<T> = Box<dyn Stream<Item = T, Error = Error> + Send + 'static>;
+pub type BoxStream<T, E = Error> = futures::stream::BoxStream<'static, Result<T, E>>;
```

Comment:
In a perfect world, we'd rename these to `BoxTryFuture` and `BoxTryStream` now, but that would be a giant diff. Maybe another day.

For bonus points, see if we can specify a lifetime parameter that defaults as `'a = 'static`.